src/ui/buttons/message.jsx

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+/** @jsx node */
+
+import { node, type ChildType } from "@krakenjs/jsx-pragmatic/src";
+
+import { CLASS } from "../../constants";
+
+const INITIAL_RESERVED_HEIGHT = "36px";
+
+type MessageProps = {|
+  messageMarkup: ?string,
+  calculatedMsgPosition: string,
+|};
+
+export function Message({
+  messageMarkup,
+  calculatedMessagePosition,
```

Comment:
Since these are in the limited context of a `<Message />` component could they be shortened to just `markup` and `position`?